README.md

```diff
@@ -14,30 +14,36 @@ an exact solution if possible or use an appropriate numeric solver otherwise.
+
+### Provisioning
+
+The container can be provisioned (created) by changing to the `docker` directory of your clone of the `nestml` Git repository and running
 
 ```
-sh nestml_docker.sh provision
+./nestml_container.sh provision
 ```
-If everything goes well, then command 'docker images' container should list a 'nestml_release' container.
 
-Currently, the container is built in the way, that is automatically processes a volume that is mounted at start up. E.g. a NESTML model (https://github.com/nest/nestml/blob/master/models/iaf_cond_alpha_implicit.nestml) can be placed into a folder called 'testing' in the home-folder. Then, the following command will execute model analysis and codegeneration for NEST target (currently, only the NEST master is supported, for earlier version use releases prio 0.1.0):
+This will download all required packages and libraries and create a container that uses the pre-built version of the [latest release of NESTML](https://github.com/nest/nestml/releases). If you are interested in using the bleeding edge version of NESTML (a.k.a. Git master), you can add the argument `--dev` to the invocation of the `./nestml_container.sh` script.
+
+If everything goes well, the list printed by the command 'docker images' should now contain the 'nestml_release' container. If you experience an error, please [open an issue](https://github.com/nest/nestml/issues) so we can look into and fix it.
+
+### Running
+
+To actually convert your model files written in NESTML to NEST C++, you have to run the Docker container. This is again done using the `nestml_container.sh` script, which for this purpos gets the command `run` as first argument and one or more folders containing one or more `.nestml` files (the folder is called `<models>` in the following description):
+
 ```
-sh nestml_docker.sh run ~/testing
+./nestml_container.sh run <models>
 ```
 
-It creates a subfolder 'build' in the 'testing' folder that contains the generated code. Per default, the module for the generated stuff is called 'testing', e.g. the name of the input folder. In order to integrate the model into nest, use the following commands (assumption, you switched to the 'testing/result'):
+This run creates a subfolder `build` in the `<models>` directory that contains the generated code and all infrastructure and source files for an extension module for NESTML, which can be dynamically loaded. The module will have the same name as the folder in which you stored the `.nestml` files. In order to compile and install the module, use the following commands:
```

Comment:
Good idea! In 35678420d370d I've added examples on a concrete usage using the real model directory as suggested by @DimitriPlotnikov below.